community/qemu/depends

```diff
@@ -0,0 +1,3 @@
+ncurses
+sdl
+make
```

Comment:
`make` can be removed. It's a given that it be installed.

Reply:
Make now removed.